Tests/IntegrationTests/PostgresNIOTests.swift

```diff
@@ -31,6 +31,25 @@ final class PostgresNIOTests: XCTestCase {
         XCTAssertNoThrow(conn = try PostgresConnection.test(on: eventLoop).wait())
         XCTAssertNoThrow(try conn?.close().wait())
     }
+    
+    func testConnectUDSAndClose() throws {
+        try XCTSkipUnless(env("POSTGRES_SOCKET") != nil)
+        var conn: PostgresConnection?
+        XCTAssertNoThrow(conn = try PostgresConnection.testUDS(on: eventLoop).wait())
+        XCTAssertNoThrow(try conn?.close().wait())
+    }
+    
+    func testConnectEstablishedChannelAndClose() throws {
+        let channel = try ClientBootstrap(group: self.group).connect(to: PostgresConnection.address()).wait()
```

Comment:
do we need
```
        try XCTSkipUnless(env("POSTGRES_SOCKET") != nil)
```
here as well?

Reply:
That's only for the UDS tests; for the established channel test I just mimic the channel bootstrap part of opening a TCP connection.